src/components/Test/index.js

```diff
@@ -1,40 +1,25 @@
 import React from 'react';
 import { connect } from 'react-redux';
-import withGraphqlService from '../../hoc/withGraphqlService';
+import withApolloService from '../../hoc/withApolloService';
 
-const Test = ({ graphqlService }) => {
+const Test = ({ client, gql }) => {
   const handler = () => {
-    graphqlService
-      .getItems(
-        `query{
-        categories{
-          categoryCode,
-          _id
-          name{
-            value
+    client
```

Comment:
I think it is better to move it to the `redux-saga` definition. Please:
1. Install `redux-saga` and setup it in application
2. Define saga and actions for fetching categories
3. Dispatch action from component

Reply:
Installed redux-saga. Task with defining saga fot test component will take another teammate